src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -56,8 +56,8 @@ def load_arguments(self, _):
         c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
         c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
         c.argument('features', features_types, options_list=['--features'], help='Space-separated list of features you want to enable.')
-        c.argument('azrbac_client_id', options_list=['--client-id'], arg_group='Azure RBAC', help='Client ID for enabling Azure RBAC. Specify when enabling azure-rbac.')
```

Comment:
>client [](start = 56, length = 6)

We need to fix the references for client in custom.py as well.. Like

telemetry.set_exception(exception='Client ID or secret is not provided for Azure RBAC', fault_type=consts.Client_Details_Not_Provided_For_Azure_RBAC_Fault,
                                    summary='Client id, client secret is required to enable/update Azure RBAC feature')

Reply:
done